test/verify/check-superuser

```diff
@@ -86,7 +87,7 @@ class TestSuperuser(MachineCase):
         b.check_superuser_indicator("Limited access")
 
         # Give us password-less sudo and use it
-        m.write("/etc/sudoers.d/admin", "admin ALL=(ALL) NOPASSWD:ALL")
+        m.write_file("/etc/sudoers.d/admin", "admin ALL=(ALL) NOPASSWD:ALL")
```

Comment:
`self`, not `m`. It has to be on teh MachineCase class so that it can hook into `.addCleanup()`.